submissions/webdevtoday/task_js_dom/index.html

```diff
@@ -0,0 +1,172 @@
+                                        </li>
+                                        <li>
+                                            <a href="#mars" class="nav-popup__item text-decoration-none">
+                                                <img src="popup-icons/mars.png" alt="mars icon" class="nav-popup__icon">
+                                                <span class="nav-popup__title">Mars</span>
+                                            </a>
+                                        </li>
+                                        <li>
+                                            <a href="#blogs" class="nav-popup__item text-decoration-none">
+                                                <img src="popup-icons/blogs.png" alt="blogs icon" class="nav-popup__icon">
+                                                <span class="nav-popup__title">Blogs</span>
+                                            </a>
+                                        </li>
+                                        <li>
+                                            <a href="#promos" class="nav-popup__item text-decoration-none">
+                                                <img src="popup-icons/promo.png" alt="promos icon" class="nav-popup__icon">
+                                                <span class="nav-popup__title">Promos</span>
+                                            </a>
+                                        </li>
+                                        <li>
+                                            <a href="#Kottans" class="nav-popup__item text-decoration-none kottans-link">Even more from Kottans</a>
+                                        </li>
+                                    </ul>
+                                </li>
+                            </ul>
+                        </div>
+                    </li>
+                    <li>
+                        <a href="#notifications" class="main-nav__item text-decoration-none">
+                            <img src="popup-icons/bell.png" alt="popup-bell" class="main-nav__icon main-nav__icon--bell">
+                        </a>
+                    </li>
+                    <li>
+                        <a href="#profile" class="main-nav__item text-decoration-none">
+                            <img src="images/user-photo.jpg" alt="user-photo" class="main-nav__icon main-nav__icon--round main-nav__icon--user-photo">
+                        </a>
+                    </li>
+                </ul>
+            </nav>
+        </header>
+        <main class="main app__main">
+            <div class="main__container">
+            </div>
+        </main>
+        <footer class="footer app__footer">
+            <div class="footer__container">
+                <span>Copyright &copy; <span data-type="year">2020</span></span>
+            </div>
+        </footer>
+    </div>
+    <script type="module" src="js/app.js"></script>
+</body>
+</html>
```

Comment:
Don't forget to add an empty line in each of the files

Reply:
Ohh, I forgot it again. I'll fix it

Reply:
I did it.